cql/CollateManagementData.cql

```diff
@@ -254,6 +254,12 @@ define MostRecentCytologyReport:
   else
     null
 
+define MostRecentCytologyReportDate:
```

Comment:
My thought was that new CQL expressions we need to add purely for the analytics capability that don't get used elsewhere in CDS would be added to Analytics.cql.